examples/data_silo/schema_content_plugin.tf

```diff
@@ -2,14 +2,9 @@ resource "transcend_data_silo" "aws" {
   type        = "amazonWebServices"
   description = "Amazon Web Services (AWS) provides information technology infrastructure services to businesses in the form of web services."
 
-  schema_discovery_plugin {
-    enabled                    = true
-    schedule_frequency_minutes = 1440 # 1 day
-    schedule_start_at          = "2022-09-06T17:51:13.000Z"
-  }
-
-  content_classification_plugin {
+  disco_class_scan_config {
     enabled                    = true
+    type                       = "FULL_SCAN"
     schedule_frequency_minutes = 1440 # 1 day
     schedule_start_at          = "2022-09-06T17:51:13.000Z"
   }
```

Comment:
`disco_class_scan_config` should replace `schema_discovery_plugin` and `content_classification_plugin`.